### ISSUE TITLE
Is it possible to add link and collapse/expand behavior to menu item?

### ISSUE THREAD

**Issue by skornev** [onokumus/metisMenu#55]
Hi, 

Here http://demo.onokumus.com/metisMenu/
If you click on item  "Menu 1.3"  it expands or collapse children items. 
Is it possible to implement that if I click on text "Menu 1.3" it will navigate to "menu13.html", but if I click on plus sign on the right corner it will just expands or collapse children items.

Thank you

**Comment by onokumus** [comment]
@skornev 
Not possible.

**Comment by nnyamekye** [comment]
is it possible to click on the menu 1.3 and it navigates me aswell as collapse the children item once the page is loaded

**Comment by eye-too** [comment]
Yes, it is possible. On many sites the parent item also represent a URL. I've tried to test it as a concept. But I am a designer, and not a strong javascript- / jQuery coder. I am sure onokumus would have found a much more elegant approach.

Test 1: http://codepen.io/eye-wonder/full/oYPeYO/
Test 2 (sub item count): http://codepen.io/eye-wonder/full/gLddOY/
